### PR TITLE
Move cookie translations, use all services & pagetypes across cypress

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Move stuff, run on all services

**Code changes:**

- removes their cypress command
- expands cookie tests to run on error pages
- Adds service config for all pages and envs
- Runs tests on all, bar known exceptions
- runs cookie banner logic tests only on news & persian

# NB: This makes the suite run in 14 mins, not 3

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
I'd marginally prefer duplication than `const visitPage` but I won't request it changed unless others agree.

We do need to add a lot of config to tests cookies and cookie banners on the many services we have on test.bbc.com though.

File: cypress/integration/specialFeatures/cookieBanner.amp.js [hunk 1]
```diff
@@ -2,9 +2,10 @@ import config from '../../support/config/services';
 import appConfig from '../../../src/app/lib/config/services';
 import describeForEuOnly from '../../support/describeForEuOnly';
 
-const filterPageTypes = (pageType, service) =>
-  config[service].pageTypes[pageType] !== undefined &&
-  pageType !== 'errorPage404';
+const serviceFilter = service => ['news', 'persian'].includes(service);
```
Comment (jamesbhobbs) [inline cypress/integration/specialFeatures/cookieBanner.amp.js:6]: Comment why these are filtered

File: cypress/integration/specialFeatures/cookieBanner.canonical.js [hunk 2]
```diff
@@ -5,12 +5,10 @@ import {
 import appConfig from '../../../src/app/lib/config/services';
 import config from '../../support/config/services';
 
-// Limited to news for now due to time test takes to run per page.
-const filterService = service => service === 'news';
+const serviceFilter = service => ['news', 'persian'].includes(service);
```
Comment (jamesbhobbs) [inline cypress/integration/specialFeatures/cookieBanner.canonical.js:9]: Comment why these are filtered

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Great work, just a tiny change req

**Review by jamesbhobbs — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
>This makes the suite run in 14 mins, not 3

Because it massively increases test coverage

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**